.github/workflows/integration_tests.yml

```diff
@@ -500,7 +500,7 @@ jobs:
             # Only the following tests are currently supported on IQM
             case $filename in
               targettests/iqm/*)
-                nvq++ -DSYNTAX_CHECK --target iqm --iqm-machine Crystal_5 $filename
+                nvq++ -DSYNTAX_CHECK --target iqm $filename
```

Comment:
I gather the new setup is that different machines will be accessible via different server urls?
If so, would we be able to test against at least two different topologies?